docs/sources/shared/visualizations/datalink-options.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Data link options
+comments: |
+  This file is used in the following visualizations: bar chart, bar gauge, candlestick, gauge, geomap, heatmap, histogram, stat, state timeline, status history, table, time series, trend, xy chart
+---
+
+Data links allow you to provide more granular context to your links. You can create links that include the series name or even the value under the cursor.
```

Comment:
I think we may want to rephrase this, I think the key value add here maybe is more around adding the ability to link to external resources / other dashboards and include contextual dashboard data / metadata - this needs some work but I think more of the general idea 😅

Reply:
I took this from the main Configure data links page, so we should probably update both now vs letting this go. Let me circle back with a proposed update.